### PR TITLE
feat(@desktop/wallet): Update network card and routing lines in SendModal

### LINKED ISSUE #8714: Update card and routing lines
Follow design under https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?node-id=9592%3A126682&t=wprMV9qSBeXQGHA9-4

Box are wider, have hover states and touting lines needs to be updated
* No clear button on the custom card, removing digits to zero is the way to clear it
*  Change the way disable works and add hoever states for the card

### Description

[comment]: # (Describe the feature you would like and include designs and Figma links if available.)
[comment]: # (Describe the use cases the implementation should cover. E.g. New users, existing users, public chat, group chat, etc)
*Summary*: ...

### Acceptance Criteria

[comment]: # (Rules for the future PR to be accepted.)

### PR DESCRIPTION
**Pull request by Khushboo-dev-cpp**
fixes #8714

### What does the PR do

Implement updated designs 
https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?node-id=9770%3A128205&t=7Rxe3LkHoLYiYRgg-4

https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?node-id=9770%3A130516&t=7Rxe3LkHoLYiYRgg-4

### Affected areas

Wallet

### StatusQ checklist

- [x] add documentation if necessary (new component, new feature)
- [x] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [x] test changes in both light and dark theme?

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

https://user-images.githubusercontent.com/60327365/209655071-4f82aca3-d02f-4506-a79b-9f741c023b37.mov

### CODE REVIEW

**Review by saledjenic — APPROVED** [review]

File: ui/StatusQ/src/StatusQ/Components/StatusCard.qml [hunk 1]
```diff
@@ -162,126 +181,178 @@ Rectangle {
...
-            }
         }
-        StatusIcon {
-            id: cardIcon
-            Layout.alignment: Qt.AlignTop | Qt.AlignRight
-            Layout.preferredHeight: 32
-            Layout.preferredWidth: 32
-            mipmap: true
+        StatusBaseText {
+            id: tertiaryText
```
Comment (stefandunca) [inline ui/StatusQ/src/StatusQ/Components/StatusCard.qml:336]: I find having the same name as the `root.tertiaryText` property confusing. How about `footerText`?

File: ui/StatusQ/src/StatusQ/Components/StatusCard.qml [hunk 2]
```diff
@@ -306,23 +377,33 @@ Rectangle {
...
+                font.weight: disabled && sensor.containsMouse ? Font.Medium : Font.Normal
             }
             PropertyChanges {
                 target: tertiaryText
                 color: Theme.palette.directColor5
             }
             PropertyChanges {
                 target: tertiaryText
-                visible: tertiaryText.text
+                visible: advancedMode ? tertiaryText.text : (!sensor.containsMouse && tertiaryText.text) || disabled
```
Comment (stefandunca) [inline ui/StatusQ/src/StatusQ/Components/StatusCard.qml:402]: When I hover and the description text disappears, it looks odd in the sandbox — wondering why this reasoning for hovering to control the visibility of the main items?

File: ui/StatusQ/src/StatusQ/Components/StatusCard.qml [hunk 3]
```diff
@@ -87,6 +92,16 @@ Rectangle {
        Used to set Tertiary text in the StatusCard
     */
     property alias tertiaryText: tertiaryText.text
+    /*!
+       \qmlproperty alias StatusCard::disableText
+       Used to set disable text in the StatusCard
```
Comment (stefandunca) [inline ui/StatusQ/src/StatusQ/Components/StatusCard.qml:97]: Not sure this best clarifies the ambiguity between `disabledText` and `disableText`. How about something on the line "Shown on hovered to indicate that click action will disable the card"?

**Review by stefandunca — APPROVED** [review]
Looks great. See some notes inline.

**Pull request merged**